README.md

```diff
@@ -0,0 +1,165 @@
+
+* **Advanced Replication Support**
+
+    Choose between asynchronous replication and synchronous replication for workloads that are sensitive to losing transactions.
+
+* **Clone**
+
+    Create new clusters from your existing clusters or backups.
+
+* **Connection Pooling**
+
+    Use [pgBouncer](https://access.crunchydata.com/documentation/postgres-operator/latest/tutorial/pgbouncer/) for connection pooling
+
+* **Affinity and Tolerations**
+
+    Have your PostgreSQL clusters deployed to Kubernetes Nodes of your preference with node affinity, or designate which nodes Kubernetes can schedule PostgreSQL instances to with Kubernetes  tolerations.
+
+* **Scheduled Backups**
+
+    Choose the type of backup (full, incremental, differential) and how frequently you want it to occur on each PostgreSQL cluster.
+
+* **Backup to S3**
+
+    Store your backups in Amazon S3 or any object storage system that supports the S3 protocol. The PostgreSQL Operator can backup, restore, and create new clusters from these backups.
+
+* **Multi-Namespace Support**
+
+    You can control how the Operator leverages Kubernetes Namespaces with several different deployment models:
+
+    * Deploy the Operator and all PostgreSQL clusters to the same namespace
+    * Deploy the Operator to one Namespace, and all PostgreSQL clusters to a different Namespace
+    * Deploy the Operator to one Namespace, and have your PostgreSQL clusters managed across multiple Namespaces
+
+* **Full Customizability**
+
+    The Operator not only makes it easy to get PostgreSQL up and running on Kubernetes-enabled platforms, but also allows you to further customize your deployments, including:
+
+    * Selecting different storage classes for your primary, replica, and backup storage
+    * Select your own container resources class for each PostgreSQL cluster deployment; differentiate between resources applied for primary and replica clusters
+    * Use your own container image repository, including `imagePullSecrets` and private repositories  support
+    * Customize your PostgreSQL configuration 
+    * Bring your own trusted certificate authority (CA) for use with the Operator API server
+    * Override your PostgreSQL configuration for each cluster
+
+## Deployment Requirements
+
+The Operator is validated for deployment on Kubernetes, GKE and EKS clusters. the Operator is cloud native and storage agnostic, working with a wide variety of storage classes, hostPath, and NFS.
+
+The Operator includes various components that get deployed to your Kubernetes cluster as shown in the following diagram and detailed in the Design section of the documentation for the version you are running.
+
+The Operator is developed and tested on CentOS and RHEL platforms but is known to run on other Linux variants.
+
+### Supported Platforms
+
+The Operator is tested against the following platforms:
+
+* Kubernetes 1.17+ 
```

Comment:
is it correct @hors ? Shall we keep only GKE and EKS?

Reply:
yes, we need to keep only GKE and EKS